### ISSUE TITLE
DDP error when fine-tuning MACE-POLAR-1 in distribution mode

### ISSUE THREAD

**Issue by legend-L24** [ACEsuit/mace#1438]
**Description of Bug:**
Hello, When I want to fine-tune  MACE-POLAR-1 model on MOF-5 training structures and the loss function only contains forces and energy. When "--distributed=True", the job fails with DDP error shown below. 

"Expected to have finished reduction in the prior iteration before starting a new one. This error indicates that your module has parameters that were not used in producing loss. You can enable unused parameter detection by passing the keyword argument `find_unused_parameters=True` to `torch.nn.parallel.DistributedDataParallel`"

However, When I set  "--distributed=False", the training works well.

**Try to fixing the bug**
In mace/cli/run_train.py, I add find_unused_parameters=True to some part, such as "distributed_model = DDP(model, device_ids=[local_rank], find_unused_parameters=True)", which can make fine-tuning successful. I am not very sure about if I provide a correct solution, so I write a report to show it.

**Comment by ilyes319** [comment]
Hey @legend-L24, i can't reproduce that, can you try the latest develop branch and tell me if works now?
